ESP32 Compile Issues

Output log from a runner:
```
/home/runner/Arduino/libraries/Adafruit_VEML7700_Library/Adafruit_VEML7700.cpp: In member function 'float Adafruit_VEML7700::readLux(luxMethod)':
  /home/runner/Arduino/libraries/Adafruit_VEML7[70](https://github.com/adafruit/Adafruit_Wippersnapper_Arduino/actions/runs/3234897613/jobs/5298588942#step:8:71)0_Library/Adafruit_VEML7700.cpp:94:10: warning: this statement may fall through [-Wimplicit-fallthrough=]
       wait = false;
       ~~~~~^~~~~~~
  /home/runner/Arduino/libraries/Adafruit_VEML7700_Library/Adafruit_VEML7700.cpp:95:3: note: here
     case VEML_LUX_NORMAL:
     ^~~~
  /home/runner/Arduino/libraries/Adafruit_VEML7700_Library/Adafruit_VEML7700.cpp:98:10: warning: this statement may fall through [-Wimplicit-fallthrough=]
       wait = false;
       ~~~~~^~~~~~~
  /home/runner/Arduino/libraries/Adafruit_VEML[77](https://github.com/adafruit/Adafruit_Wippersnapper_Arduino/actions/runs/3234897613/jobs/5298588942#step:8:78)00_Library/Adafruit_VEML7700.cpp:99:3: note: here
     case VEML_LUX_CORRECTED:
     ^~~~
```

--- a/Adafruit_VEML7700.cpp
+++ b/Adafruit_VEML7700.cpp
@@ -89,13 +89,13 @@ bool Adafruit_VEML7700::begin(TwoWire *theWire) {
  */
 float Adafruit_VEML7700::readLux(luxMethod method) {
   bool wait = true;
-  switch (method) {
-  case VEML_LUX_NORMAL_NOWAIT:
+
+  if (method == VEML_LUX_NORMAL_NOWAIT || method == VEML_LUX_CORRECTED_NOWAIT)
     wait = false;
+
+  switch (method) {
   case VEML_LUX_NORMAL:
     return computeLux(readALS(wait));
-  case VEML_LUX_CORRECTED_NOWAIT:
-    wait = false;
   case VEML_LUX_CORRECTED:
     return computeLux(readALS(wait), true);
   case VEML_LUX_AUTO:

--- a/library.properties
+++ b/library.properties
@@ -1,5 +1,5 @@
 name=Adafruit VEML7700 Library
-version=2.1.1
+version=2.1.2
 author=Adafruit
 maintainer=Adafruit <info@adafruit.com>
 sentence=Arduino library for the VEML7700 sensors in the Adafruit shop

